tests/test_pfcwd.py

```diff
@@ -0,0 +1,221 @@
+
+def setPfcStormQueueOid(dvs, queue_oid):
+   cnt_r = redis.Redis(unix_socket_path=dvs.redis_sock, db=swsscommon.COUNTERS_DB,
+                       encoding="utf-8", decode_responses=True)
+   cnt_r.hset("COUNTERS:"+queue_oid, "DEBUG_STORM","enabled")
+   import pdb;pdb.set_trace()
+
+def get_acl_table_id(self, dvs):
+    tbl = swsscommon.Table(self.adb, "ASIC_STATE:SAI_OBJECT_TYPE_ACL_TABLE")
+    keys = tbl.getKeys()
+
+    for k in  dvs.asicdb.default_acl_tables:
+        assert k in keys
+
+    acl_tables = [k for k in keys if k not in dvs.asicdb.default_acl_tables]
+    if len(acl_tables) == 1:
+        return acl_tables[0]
+    else:
+        return None
+
+def verify_acl_group_num(self, expt):
+    atbl = swsscommon.Table(self.adb, "ASIC_STATE:SAI_OBJECT_TYPE_ACL_TABLE_GROUP")
+    acl_table_groups = atbl.getKeys()
+    assert len(acl_table_groups) == expt
+
+    for k in acl_table_groups:
+        (status, fvs) = atbl.get(k)
+        assert status == True
+        for fv in fvs:
+            if fv[0] == "SAI_ACL_TABLE_GROUP_ATTR_ACL_STAGE":
+                assert fv[1] == "SAI_ACL_STAGE_INGRESS"
+            elif fv[0] == "SAI_ACL_TABLE_GROUP_ATTR_ACL_BIND_POINT_TYPE_LIST":
+                assert fv[1] == "1:SAI_ACL_BIND_POINT_TYPE_PORT"
+            elif fv[0] == "SAI_ACL_TABLE_GROUP_ATTR_TYPE":
+                assert fv[1] == "SAI_ACL_TABLE_GROUP_TYPE_PARALLEL"
+            else:
+                assert False
+
+def verify_acl_group_member(self, acl_group_ids, acl_table_id):
+    atbl = swsscommon.Table(self.adb, "ASIC_STATE:SAI_OBJECT_TYPE_ACL_TABLE_GROUP_MEMBER")
+    keys = atbl.getKeys()
+
+    member_groups = []
+    for k in keys:
+        (status, fvs) = atbl.get(k)
+        assert status == True
+        assert len(fvs) == 3
+        for fv in fvs:
+            if fv[0] == "SAI_ACL_TABLE_GROUP_MEMBER_ATTR_ACL_TABLE_GROUP_ID":
+                assert fv[1] in acl_group_ids
+                member_groups.append(fv[1])
+            elif fv[0] == "SAI_ACL_TABLE_GROUP_MEMBER_ATTR_ACL_TABLE_ID":
+                assert fv[1] == acl_table_id
+            elif fv[0] == "SAI_ACL_TABLE_GROUP_MEMBER_ATTR_PRIORITY":
+                assert True
+            else:
+                assert False
+
+    assert set(member_groups) == set(acl_group_ids)
+
+def verify_acl_port_binding(self, dvs, bind_ports):
+    atbl = swsscommon.Table(self.adb, "ASIC_STATE:SAI_OBJECT_TYPE_ACL_TABLE_GROUP")
+    acl_table_groups = atbl.getKeys()
+    assert len(acl_table_groups) == len(bind_ports)
+
+    atbl = swsscommon.Table(self.adb, "ASIC_STATE:SAI_OBJECT_TYPE_PORT")
+    port_groups = []
+    for p in [dvs.asicdb.portnamemap[portname] for portname in bind_ports]:
+        (status, fvs) = atbl.get(p)
+        for fv in fvs:
+            if fv[0] == "SAI_PORT_ATTR_INGRESS_ACL":
+                assert fv[1] in acl_table_groups
+                port_groups.append(fv[1])
+
+    assert len(port_groups) == len(bind_ports)
```

Comment:
Take a look at `dvs_acl.py`, I think these methods are already implemented for you.